sagemaker-pipeline-multi-model/sagemaker-pipeline/restate-athena-california.flow

```diff
@@ -25,17 +25,12 @@
       },
       "trained_parameters": {
         "query_execution_id": "ca8a88de-b2b8-403b-b841-f3a9c1a750a0",
-        "ctas_table_name": "sagemaker_tmp_5f97be7f253044558c96b79e9a8fad40",
-        "ctas_s3_output_location": "s3://sagemaker-ap-southeast-1-240964962523/athena/5f97be7f253044558c96b79e9a8fad40/data/"
+        "ctas_table_name": "sagemaker_tmp_5f97be7f253044558c96b79e9a8fad40"
```

Comment:
Does the customer need to fill in their own table ID?

Reply:
[The flow file is exported from Data Wrangler](https://docs.aws.amazon.com/sagemaker/latest/dg/data-wrangler-data-flow.html). The import of data, along with the transformations, were done from SageMaker Studio and then the .flow file was exported. the ctas_table_name is already populated upon export of the .flow file. Thank you!